logbook-core/src/main/java/org/zalando/logbook/DefaultLogbook.java

```diff
@@ -56,7 +56,7 @@
                 final Correlation<HttpRequest, HttpResponse> correlation =
                         new SimpleCorrelation<>(correlationId, duration, request, response);
                 final String message = formatter.format(correlation);
-                writer.writeResponse(new SimpleCorrelation<>(correlationId, duration, format, message));
```

Comment:
Or extend `Correlation` interface to expose `getOriginalRequest` and `getOriginalResponse`